src/handler-builder.ts

```diff
@@ -35,6 +35,14 @@ import * as remoteConfig from './providers/remoteConfig';
 import * as storage from './providers/storage';
 import * as testLab from './providers/testLab';
 
+/**
+ * The `HandlerBuilder` class facilitates the writing of functions by producers
+ * of Firebase Extensions and developers that want to use gcloud CLI or Cloud Console
+ * to deploy their functions.
```

Comment:
I didn't know that these were also used for gcloud CLI and Cloud Console?

Reply:
It was interesting news to me too!  This addition was requested by a knowledgeable SWE in an earlier review.